crates/services/src/services/filesystem_watcher.rs

```diff
@@ -152,7 +153,9 @@ pub fn async_watcher(
     Ok((debouncer, rx, canonical_root))
 }
 
+#[allow(dead_code)]
 async fn async_watch<P: AsRef<Path>>(path: P) -> Result<(), FilesystemWatcherError> {
+#[allow(dead_code)]
```

Comment:
Resolved: `async_watch` is cfg‑gated for tests or `dev-utils` feature; added `dev-utils` feature in services/Cargo.toml; removed stray dead_code on let binding and kept one on the function to satisfy `-D warnings` when feature is enabled.